rmf_traffic_ros2/include/rmf_traffic_ros2/schedule/Negotiation.hpp

```diff
@@ -68,6 +68,43 @@ class Negotiation
   /// Get the current timeout duration setting.
   rmf_traffic::Duration timeout_duration() const;
 
+  using TableViewPtr = rmf_traffic::schedule::Negotiation::Table::ViewerPtr;
+  using StatusUpdateCallback =
+    std::function<void (uint64_t conflict_version, TableViewPtr table_view)>;
+  
+  /// Register a callback with this Negotiation manager that triggers
+  /// on negotiation status updates.
+  ///
+  /// \param[in] cb
+  ///   The callback function to be called upon status updates.
+  void on_status_update(StatusUpdateCallback cb);
+
+  using StatusConclusionCallback =
+    std::function<void (uint64_t conflict_version, bool success)>;
+  
+  /// Register a callback with this Negotiation manager that triggers
+  /// on negotiation status conclusions.
+  ///
+  /// \param[in] cb
+  ///   The callback function to be called upon status conclusions.
+  void on_conclusion(StatusConclusionCallback cb);
+
+  /// Get a Negotiation::TableView that provides a view into what participants are
+  /// proposing.
+  ///
+  /// This function does not care about table versioning.
+  ///
+  /// \sa find()
```

Comment:
I think this `\sa find()` was probably copy/pasted from the `rmf_traffic::schedule::Negotiation` documentation, because there doesn't seem to be a `find(~)` function in this class.

Reply:
fixed documentation 94509f9